lib/ex_aws/dynamo.ex

```diff
@@ -364,6 +406,25 @@ defmodule ExAws.Dynamo do
   defp do_convert_billing_mode(attributes, value) when is_list(attributes),
     do: Keyword.replace!(attributes, :billing_mode, value)
 
+  defp add_table_opt({:stream_enabled, enabled}, data) do
+    data
+    |> ensure_entry("StreamSpecification")
+    |> put_in(["StreamSpecification", "StreamEnabled"], enabled)
+  end
+
+  defp add_table_opt({:stream_view_type, type}, data) do
```

Comment:
The value of `type` is "enforced" by the typespec (insofar as that is actually enforcement). Just testing against dynamo-local, it doesn't seem to return an error, but I suspect it might have a very lightweight implementation of streams (which is to say, no implementation at all beyond accepting the option). I'm with you, though, that this is really for DDB to accept or reject. I don't think there's value in us adding a layer of checking which we then have to maintain.

Reply:
👍